examples/weather_service/module/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "weather_service"
+version = "0.1.0"
+authors = ["Ivan Petrov <ivanpetrov@google.com>"]
+edition = "2018"
+license = "Apache-2.0"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+anyhow = "*"
+log = "*"
+oak = "=0.1.0"
```

Comment:
Could you enable the `linear-handles` feature for this new example? We will soon switch to that as the default behaviour 😄

Reply:
The plan is for this example to be built on the oak_functions ABI, not the current oak ABI, so I think you can ignore this :) @ipetr0v could you confirm?

Reply:
Yes, this example will only work on `oak_functions` and is not related to `oak_runtime`